modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2QueryInfo.java

```diff
@@ -106,22 +119,24 @@ public long time() {
      * @param additionalInfo Additional query info.
      */
     public void printLogMessage(IgniteLogger log, String msg, String additionalInfo) {
-        StringBuilder msgSb = new StringBuilder(msg + " [");
-
-        if (additionalInfo != null)
-            msgSb.append(additionalInfo).append(", ");
-
-        msgSb.append("duration=").append(time()).append("ms")
-            .append(", type=").append(type)
-            .append(", distributedJoin=").append(distributedJoin)
-            .append(", enforceJoinOrder=").append(enforceJoinOrder)
-            .append(", lazy=").append(lazy)
-            .append(", schema=").append(schema);
+        StringBuilder msgSb = new StringBuilder(msg);
 
-        msgSb.append(", sql='")
-            .append(sql);
+        if (queryId == RunningQueryManager.UNDEFINED_QUERY_ID)
+            msgSb.append(" [globalQueryId=(undefined), node=").append(nodeId);
+        else
+            msgSb.append(" [globalQueryId=").append(QueryUtils.globalQueryId(nodeId, queryId));
```

Comment:
let's change signature of QueryUtils.globalQueryId(nodeId, qruId) from Long for qryId to primitive